protocol.md

```diff
@@ -23,6 +23,7 @@ Name            | Type   | Required | Authenticated
 SERIALIZED_BODY | bytes  | Yes      | Yes
 PAYLOAD_TYPE    | string | Yes      | Yes
 KEYID           | string | No       | No
+EXTENSION       | object | No       | No
```

Comment:
What does parameter mean here and how does it relate to the SIGNATURE definition above? Is EXTENSION really a parameter or an implementation detail of SIGNATURE?

Reply:
I think it's the latter but I chose to include this given KEYID is also here. Perhaps we need to restructure these?

Reply:
Okay, I'm okay with adding it. Would you mind writing an issue?

Reply:
https://github.com/secure-systems-lab/dsse/issues/66